frontend/src/employee-frontend/components/messages/MessageEditor.tsx

```diff
@@ -190,13 +190,29 @@ export default React.memo(function MessageEditor({
       const nestedAccount = nestedAccounts.find(
         (account) => account.account.id === message.sender.value
       )
-      if (nestedAccount && !isNestedGroupMessageAccount(nestedAccount)) {
-        setReceiverTree(availableReceivers)
-      } else if (nestedAccount && isNestedGroupMessageAccount(nestedAccount)) {
-        const groupId = nestedAccount.daycareGroup.id
-        const selection = getSubTree(availableReceivers, groupId)
-        if (selection) {
-          setReceiverTree(selection)
+      if (!nestedAccount) {
+        throw 'Invalid sender id'
+      } else {
```

Comment:
Throwing a plain string is not a good practice even though it is allowed by the language. You should throw an error with a good error message instead for debuggability. The `else` is redundant after early return.